src/components/MiradorMenuButton.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+
+
+/**
+ * MiradorMenuButton ~ Wrap the given icon prop in an IconButton and a Tooltip.
+ * This shares the passed in aria-label w/ the Tooltip (as title) and the IconButton
+ * All props besides icon are spread to the IconButton component
+*/
+export function MiradorMenuButton(props) {
+  const { 'aria-label': ariaLabel, disabled } = props;
+  const { icon, ...iconButtonProps } = props;
+  return (
+    <Tooltip title={ariaLabel}>
+      {
+        disabled // Wrap IconButton in span if it is disabled
+          ? <span><IconButton {...iconButtonProps}>{icon}</IconButton></span>
+          : <IconButton {...iconButtonProps}>{icon}</IconButton>
```

Comment:
`IconButton` also has a disabled property. Should we be passing that through to the button when it's in the disabled state?